pages/tools/cli.tsx

```diff
@@ -164,7 +164,7 @@ export default function CliPage() {
                       codeBlocks={[
                         {
                           language: 'npm',
-                          code: 'asyncapi new'
+                          code: 'asyncapi new file'
```

Comment:
@Shurtu-gal Can you please confirm this change?

Reply:
Yeah, this is correct.